src/styles/control.styl

```diff
@@ -0,0 +1,6 @@
+//Custom controls
```

Comment:
Instead of variables, it would be better to create a mixin that sets the `height`, `min-width` and `box-sizing` rules. That way, one could just run the mixin as follows.

```styl
.vocab.component
  control()
```

Reply:
```suggestion
@mixin control {
height: 2.25em;
width: 14.5em;
box-sizing: border-box;
}

```